opendatahub-io/odh-dashboard
# Adds 'Area' support for TrustyAI and Performance Metric feature sets

Closes: #2088 
## Description


Contains the following updates:
* Merges area features from main
* Updates TrustyAI feature set to use area api
* Updates Performance metrics features to use area api

## How Has This Been Tested?



* Deploy on cluster
* Verify TrustyAI features are disabled by either flag or DSC
* Verify Performance Metrics features are disabled by either flag or DSC

Image: quay.io/acreasy/odh-dashboard:issue-2088 can be used for deployed test.

## Test Impact


N/a

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
